### DISCUSSION TITLE
Problem "save config" in configuration standard

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#3551] [Questions & Answers] [answered]
I have a strange problem in System>Configuration>"Standard Configuration"/Shipping Methods.

If I change the various settings and then try to save the "Save Config" button doesn't work.

However, if I change from "Standard configuration" to "Web-Site" the "Save Config" button works.

OpenMage 20.1.1

**Comment by kiatng** [comment]
I have 20.1.1, I cannot replicate the issue.

**Reply by Danieleeffe1** [reply]
I have installed some plug-ins, I will investigate by uninstalling them backwards.

I will inform the community about the cause (if I find it)

**Reply by Danieleeffe1** [reply]
I did a test

to have a demo of the MIMIC template ready to show to the community I had installed the old Magento1 example files.

I decided to delete the Local.xml file and reinitialize an OpenMage installation without uninstalling the plug-ins

Now the "Save Config" button works

**Comment by addison74** [comment]
In such situations I would look in the browser console. If there are problems with the JS scripts, it is impossible not to show them. Also, take a look into the log files /var/log/....

As a tip, I always have a reliable OpenMage instance (based on DDEV) from which I start with a new test version. If I get into trobule then it is obvious that it comes from what I did later. In general, most people tend not to keep a record of the changes and this is the source of the problems. 

The other day I touched a few servers where strange things were happening. Luckily they all used the git based etckeeper package. I found a lot of changes that were from manual edits and not from package updates. When I asked those who had root rights on the machines, they all shrugged their shoulders.

**Reply by Danieleeffe1** [reply, marked as answer]
I found that the problem lies in setting the shipping methods.

by default shipping methods are enabled in sample data
UPS
DHL
USPS
FedEx

One of these enabled settings prevents FlatTax from being activated
